src/main/java/org/mule/module/json/internal/ValidateJsonSchemaOperation.java

```diff
@@ -131,7 +133,9 @@ public void stop() {
   @Validator
   @Execution(CPU_INTENSIVE)
   @Throws(SchemaValidatorErrorTypeProvider.class)
-  public void validateSchema(@Summary("The schema location") @Path(type = FILE, acceptedFileExtensions = "json") String schema,
+  public void validateSchema(@Optional @Summary("The schema location") @Path(type = FILE,
+      acceptedFileExtensions = "json") String schema,
+                             @Optional @Summary("The schema content to validate") @Expression(ExpressionSupport.SUPPORTED) @DisplayName("Schema Content") String schemaContent,
```

Comment:
For this schemaContent, this field should be a big text box for user to input the whole schema.
But here looks like it's still a field.
How did you test when you input your schema?

Reply:
I added the annotation @Text for the field schemaContent.